exporters/otlp/protocoldriver.go

```diff
@@ -66,16 +85,26 @@ type splitDriver struct {
 	trace  ProtocolDriver
 }
 
+// noopDriver implements the ProtocolDriver interface and
+// is used internally to implement split drivers that do not have
+// all drivers configured.
+type noopDriver struct{}
```

Comment:
```suggestion
type noopDriver struct{}

var _ ProtocolDriver = (*noopDriver)(nil)
```

Assert that the `noopDriver` satisfies the `ProtocolDriver` interface.